man/man5/zfs-module-parameters.5

```diff
@@ -1765,6 +1765,20 @@ table.
 Default value: \fB131,072\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBfletcher_4_impl_set\fR (string)
+.ad
+.RS 12n
+Select a fletcher 4 implementation to use.
+Two selectors are supported: \fBfastest\fR and \fBoriginal\fR. If there exists
```

Comment:
How about renaming the "original" -> "scalar". Note that for raidz, I had to rewrite the 'original' code for compatibility and performance reasons, and I named 'scalar' (the name usually used for normal CPU code when vector implementations are also around). I also hope that 'original' can be removed altogether at some time. Here, it's unlikely that the original, scalar, code can be further improved.

Also, the parameter should support selecting all available implementations + 'fastest', i.e. :
scalar, avx2, fastest